### PR TITLE
ms sql datasource

### PR DESCRIPTION
**Pull request by maximlopin**
https://github.com/mindsdb/mindsdb_native/issues/39

### CODE REVIEW

**Comment by maximlopin** [comment]
tests fail on [ubuntu, python3.8] with error `AttributeError: module 'platform' has no attribute 'linux_distribution' `
and on OSX it fails to compile: `src/_mssql.c:607:10: fatal error: 'sqlfront.h' file not found `

**Review by George3d6 — CHANGES_REQUESTED** [review]
@maximlopin 

See: https://github.com/pymssql/pymssql/issues/586

consider using another library as this one is currently not being actively maintained.

File: .travis.yml [hunk 1]
```diff
@@ -57,6 +57,7 @@ install:
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then travis_wait 15 pip3 install --user --no-cache-dir -e .; sleep 10s; fi
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then travis_wait 15 pip3 install --user --no-cache-dir -r requirements_test.txt; fi
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then travis_wait 15 pip3 install --user --no-cache-dir -r optional_requirements_extra_data_sources.txt; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then travis_wait 15 sudo apt-get install unixodbc-dev; fi
```
Comment (btseytlin) [inline .travis.yml]: Will it work on mac and win?
Perhaps there is a pure-python connector that we can use?

Comment (btseytlin) [reply .travis.yml]: From a quick google search I found these:
* https://github.com/jiangwen365/pypyodbc
* https://github.com/denisenkom/pytds

**Review by btseytlin — DISMISSED** [review]

**Review by George3d6 — APPROVED** [review]

**Pull request merged**